test/loader-common.js

```diff
@@ -105,12 +105,15 @@ export const LoaderCommonSettings = function(settings) {
  *        Function to be run in the beforeEach after loader creation. Takes one parameter,
  *        the loader for custom modifications to the loader object.
  */
-export const LoaderCommonFactory = (
+
+export const LoaderCommonFactory = ({
   LoaderConstructor,
   loaderSettings,
   loaderBeforeEach,
-  usesAsyncAppends = true
-) => {
+  usesAsyncAppends = true,
+  initSegments = true,
```

Comment:
This was refactored since IE and edge cannot parse `muxedSegment` or `video/audioInitSegment` as `utf8` which was causing the segment loader to die in vtt segment loader. Now we: 
1. Do not run initSegment tests for vtt segment loader, as vtt-segment loader doesn't ever deal with init segments.
2. We use real vtt data instead of muxed segment for the vtt tests.